common-content/en/module/js3/capturing-events/index.md

```diff
@@ -27,33 +27,13 @@ function handleSearchInput(event) {
 }
 ```
 
-When the "input" event fires, our handler function will run. Inside the handler we can access the updated input value: `const searchTerm = event.target.value;`
-
-So our key steps are:
-
-1. Add an input event listener to the search box
-2. In the handler, get `value` of input element
-3. Set the new state based on this value.
-4. Call our `render` function again.
-
-{{<note type="warning" title="One thing at a time!">}}
-But we're not going to do all of these at once! Stop and implement just the first two steps (adding the event listener, and getting the value), and `console.log` the search term.
-
-{{</note>}}
-
-We will make sure this works before we try to change the UI. Why? If we try to add the event listener and something _doesn't_ work, we will only have a little bit of code to debug.
-
-If we tried to solve the whole problem (updating the UI) and something didn't work, we would have a _lot_ of code to debug, which is harder!
-
-We've now demonstrated that we can capture search text on every keystroke:
+These listeners wait for specific events to occur, and when they do, they trigger a callback function we've defined. This gives us a way to make our code respond to user actions rather than running all at once.
 
 ```js
-const searchBox = document.getElementById("search");
+const search = document.getElementById("search");
+search.addEventListener("input", handleInput);
+```
 
-searchBox.addEventListener("input", handleSearchInput);
+When we call `addEventListener`, it doesn't immediately execute the `handleInput` function. Instead, it sets up a listener that will run this function later. Event listeners are actually part of the Event API provided by web browsers - they aren't part of the core JavaScript language! When you create an event listener, you're essentially making a request to a Web API to handle this functionality for you. In this pattern, the callback function (`handleInput`) only executes when a user types. We're using functions as containers for code that needs to execute at a later time, specifically in response to user interactions. This approach lets us tell the browser exactly what actions to take once a particular event occurs.
```

Comment:
essentially? functionality? Can you do a Hemingway pass? 🙏

Reply:
Yeah no problem - I've simplified a lot of the language down with Hemmingway